### PR TITLE
Validate form

### PR DESCRIPTION
**Pull request by oovillagran**
**Portfolio: validate contact form**

For this milestone of my portfolio website, we implemented a client-side validation of the data in the contact form before submission. I used the figma template #1.

The general requirements of this project are:

- There are no linter errors. 
- Used correct GitHub flow.
- I documented my work in a professional way.

**The project requirements are:**

- [ ] I implemented a simple validation:
     - The content of the email field has to be in lower case.
- [ ] I implemented the following interactions:
     - When the user submits the form, I check if the email input is in lower case.
     - If the validation is OK, the form is sent.
     - If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @oovillagran, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project, that doesn't have a pull request description, kindly add one. See a demo on how to add a PR description below:

pull_request_desc
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by oovillagran** [comment]
Hi @topeogunleye,

I'm sorry, I forget to put the PR description.

File: script.js [hunk 1]
```diff
@@ -247,3 +247,25 @@ sButton.forEach((button, index) => {
...
+
+const form = document.getElementById('contact_form');
+
+function clientValidationByEmail() {
+  const email = document.getElementById('email');
+
+  if (email.value !== email.value.toLowerCase()) {
+    alert('E-mail must be in lower case 😉');
+    return false;
+  }
```
Comment (AmaduKamara) [inline script.js]: - [ ] From the lessons previously given to you, it's required that we follow [JavaScript Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and avoid using alert() and others you will find in that link. Kindly display an error text message close to the button when there is an uppercase letter in the email just as the requirement instructed and not alert(). 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @hafedEfheij,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!z

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Validation check is implemented as expected for email 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: script.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply script.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @oovillagran  @emarkees  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thankyou @Abenezer-Tilahun. I appreciate your kind review. 👌

**Pull request merged**